.github/workflows/linting.yml

```diff
@@ -80,6 +81,7 @@ jobs:
                 - '**.js'
                 - '**.jsx'
                 - '**.ts'
+                - '**.svelte'
```

Comment:
Do we have any svelte files in `tools/eslint-excludelist.json`, or plan to ever add any?

The ideal is that that's a remove-only list, and eventually it'll be empty.

Reply:
I can't think of a situation where this would be needed for Boost

Reply:
Could be worth dropping this line from the PR then. But the only thing it hurts is that it might make some checks run when they're not really needed.

Reply:
Sorry, I actually did not pay proper attention to the original feedback. You are right, no need to have it there at all then. I removed it in commit b6d3942